### PR TITLE
fix: NFT events change reintroduced

### LINKED ISSUE #539: Bug: BSX Rococo is missing event data
Following features are missing / are not implemented on the Basilisk Rococo instance:

- #416
- #457 

pls @lumir-mrkva @green-jay 🔝 📦 ⛓️

### PR DESCRIPTION
**Pull request by enthusiastmartin**
Updates to latest warehouse version.

Fixes #539

### CODE REVIEW

File: runtime/basilisk/src/lib.rs [hunk 1]
```diff
@@ -351,7 +351,7 @@ parameter_types! {
 
 impl pallet_transaction_multi_payment::Config for Runtime {
 	type Event = Event;
-	type AcceptedCurrencyOrigin = MajorityTechCommitteeOrRoot;
+	type AcceptedCurrencyOrigin = EnsureRoot<AccountId>;
```
Comment (lumir-mrkva) [inline runtime/basilisk/src/lib.rs:354]: i suspect this change is not related to the PR

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]

File: runtime/basilisk/src/lib.rs (hunk 1 above)
Comment (enthusiastmartin) [reply runtime/basilisk/src/lib.rs:354]: Oh. Missed that. I was explaining origin concept to audit guys. Forgot to revert.

Comment (enthusiastmartin) [reply runtime/basilisk/src/lib.rs]: reverted

**Review by lumir-mrkva — APPROVED** [review]

**Pull request merged**